atlasexec/atlas.go

```diff
@@ -318,19 +322,31 @@ func (c *Client) MigrateLint(ctx context.Context, params *MigrateLintParams) (*S
 	if params.Writer != nil || params.Web {
 		return nil, errors.New("atlasexec: Writer or Web reporting are not supported with MigrateLint, use MigrateLintError")
 	}
-	r, err := c.runCommand(ctx, lintArgs(params), validJSON)
-	return jsonDecode[SummaryReport](r, err)
+	return jsonDecode[SummaryReport](c.runCommand(ctx, lintArgs(params), validJSON))
 }
 
+// ErrLint is returned by MigrateLintError when linting errors are detected
+var ErrLint = errors.New("atlasexec: lint errors exist")
+
 // MigrateLintError runs the 'migrate lint' command, the output is written to params.Writer
 func (c *Client) MigrateLintError(ctx context.Context, params *MigrateLintParams) error {
-	r, err := c.runCommand(ctx, lintArgs(params))
+	r, exitCode, err := c.runCommand(ctx, lintArgs(params))
 	if err != nil {
 		return err
 	}
+	if exitCode > 1 {
```

Comment:
>0?

Reply:
exit code of 0 is considered non-error, or am I missing something?